src/admin/components/UploadImage.vue

```diff
@@ -1,6 +1,9 @@
 <template>
     <div class="dokan-upload-image" @click="uploadImage">
-        <img v-if="! showButton" :src="image.src ? image.src : src">
+        <div class="dokan-upload-image-container">
+            <img v-if="! showButton" :src="image.src ? image.src : src">
```

Comment:
Bhai, please consider using the `if condition` to the image uploader wrapper `div` in the previous line.